tests/manual/tests.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import platform
+from subprocess import call
+
+def main():
+    doxyfileName = "Doxyfile"
+    if platform.system() == "Windows":
+        print ("Windows")
+        doxyfileName = "Doxyfile.windows"
```

Comment:
If I am not mistaken, the only difference between Doxyfile and Doxyfile.windows is the installation path of Doxyqml in `FILTER_PATTERNS`. If so then it would be better to just generate a Doxyfile with the correct path, especially since nothing guarantees that Python is installed in `C:\Python3`.